### DISCUSSION TITLE
Omega layers html5 in markdown?

### DISCUSSION THREAD

**Discussion by CodingNepal14** [Xbanjwk17/UserCommunity#15] [Ideas]
<html lang="en"><head>
    <meta charset="UTF-8">
    <title>Ω-Lλγers: 0 α</title>
    <link rel="icon" href="favicon.png">

    <meta name="keywords" content="idle, incremental, omega, layers, vepro, veprogames">
    <meta name="description" content="An Incremental Game with unlimited Layers of Prestige">
    <meta name="url" content="https://veprogames.github.io/omega-layers/">
    <meta name="image" content="https://veprogames.github.io/omega-layers/cover.png">
    <meta name="theme-color" content="#f2ae3a">

    <meta name="og:title" content="Omega Layers">
    <meta name="og:type" content="website">
    <meta name="og:description" content="An Incremental Game with unlimited Layers of Prestige">
    <meta name="og:url" content="https://veprogames.github.io/omega-layers/">
    <meta name="og:image" content="https://veprogames.github.io/omega-layers/cover.png">

    <meta name="twitter:card" content="summary_image_large">
    <meta name="twitter:title" content="Omega Layers">
    <meta name="twitter:description" content="An Incremental Game with unlimited Layers of Prestige">
    <meta name="twitter:image" content="https://veprogames.github.io/omega-layers/cover.png">
    <meta name="twitter:site" content="@veprogames">
    <meta name="twitter:creator" content="@veprogames">

    <link rel="stylesheet" href="css/main.css">
    <link id="theme" rel="stylesheet" href="css/themes/dark.css">
</head>
<body>
<div id="loading" style="opacity: 0;">
    <h2><span class="omega">Ω</span> - L λ γ e r s</h2>
    <p>Loading Savegame...</p>
</div>
<div id="app"><header><nav><div><h2><span class="omega">Ω</span> - L λ γ e r s</h2> <button><img alt="L" src="images/layers.svg" class="inline"> Layers</button> <button><img alt="A" src="images/star.svg" class="inline"> Achievements</button> <button><img alt="S" src="images/information-circle.svg" class="inline"> Guide</button> <button><img alt="S" src="images/settings.svg" class="inline"> Settings</button></div> <div><button disabled="disabled"><span>Reach <span class="resource-name" style="color: rgb(140, 129, 115); text-shadow: currentcolor 0px 0px 0.025em, currentcolor 0px 0px 0.0125em, currentcolor 0px 0px 0em;">β</span></span></button> <button disabled="disabled"><span>Reach <span class="resource-name" style="color: rgb(148, 153, 102); text-shadow: currentcolor 0px 0px 0.05em, currentcolor 0px 0px 0.025em, currentcolor 0px 0px 0em;">γ</span></span></button> <button disabled="disabled"><span>Reach <span class="resource-name" style="color: rgb(116, 166, 89); text-shadow: currentcolor 0px 0px 0.075em, currentcolor 0px 0px 0.0375em, currentcolor 0px 0px 0em;">δ</span></span></button> <button disabled="disabled"><span>Reach <span class="resource-name" style="color: rgb(231, 13, 242); text-shadow: currentcolor 0px 0px 0.225em, currentcolor 0px 0px 0.1125em, currentcolor 0px 0px 0em;">κ</span></span></button></div></nav> <div class="news-ticker"><span style="animation-duration: 13.7s;">Suggest more messages in the Discord!</span></div> <div class="layer-navigation"><button title="alpha" class="" style="font-size: 2.5rem;"><span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span> </button></div></header> <main><div class="notification-container"></div> <div><div class="prestige-layer"><p class="resource-display">You have <span>0</span> <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></p>  <div class="resource-button"><button>+1 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></div>  <div class="tabs"><button>Generators</button> <button>Upgrades</button>    <button>Statistics</button> <button>Max All (M)</button></div> <div><table class="generator-table"><thead><th>Generator</th> <th>Amount</th> <th>Buy</th> <th>Buy Until 10</th></thead> <tr><td>Generator <span style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α<sub>1</sub></span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">10 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td> <td><button disabled="disabled">100 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td></tr><tr><td>Generator <span style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α<sub>2</sub></span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">100 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td> <td><button disabled="disabled">1,000 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td></tr><tr><td>Generator <span style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α<sub>3</sub></span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">1,000 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td> <td><button disabled="disabled">10,000 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td></tr><tr><td>Generator <span style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α<sub>4</sub></span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">10,000 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td> <td><button disabled="disabled">100,000 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td></tr><tr><td>Generator <span style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α<sub>5</sub></span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">1.00e6 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td> <td><button disabled="disabled">1.00e7 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td></tr><tr><td>Generator <span style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α<sub>6</sub></span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">1.00e8 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td> <td><button disabled="disabled">1.00e9 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td></tr><tr><td>Generator <span style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α<sub>7</sub></span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">1.00e10 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td> <td><button disabled="disabled">1.00e11 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td></tr><tr><td>Generator <span style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α<sub>8</sub></span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">1.00e13 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td> <td><button disabled="disabled">1.00e14 <span class="resource-name" style="color: rgb(128, 128, 128); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">α</span></button></td></tr></table></div>     </div></div>       </main> <footer><div><a href="https://discord.gg/75d7Jj5" target="_blank">Discord</a> <a href="https://www.cook1eegames.feedia.co" target="_blank">Website</a></div></footer></div>

<script src="https://polyfill.io/v3/polyfill.min.js?features=Array.prototype.flat"></script>

<script src="js/break_eternity.min.js"></script>
<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/@antimatter-dimensions/notations@1.6.0"></script>

<script src="https://cdn.jsdelivr.net/npm/vue@2"></script>

<script src="js/vocabulary.js"></script>
<script src="js/classes/random.js"></script>
<script src="js/classes/utils.js"></script>
<script src="js/classes/featureunlockmanager.js"></script>

<script src="js/classes/generator.js"></script>
<script src="js/classes/powergenerator.js"></script>
<script src="js/classes/upgrade.js"></script>
<script src="js/classes/challenge.js"></script>
<script src="js/classes/prestigelayer.js"></script>
<script src="js/classes/alephlayer.js"></script>
<script src="js/classes/achievement.js"></script>
<script src="js/classes/automator.js"></script>
<script src="js/classes/restacklayer.js"></script>
<script src="js/classes/metalayer.js"></script>
<script src="js/classes/notification.js"></script>

<script src="js/components/layer-colored-text.js"></script>
<script src="js/components/layer-locked-button.js"></script>
<script src="js/components/generator.js"></script>
<script src="js/components/generator-table.js"></script>
<script src="js/components/resource-name.js"></script>
<script src="js/components/resource-display.js"></script>
<script src="js/components/upgrade.js"></script>
<script src="js/components/resource-upgrade.js"></script>
<script src="js/components/aleph-upgrade.js"></script>
<script src="js/components/restack-upgrade.js"></script>
<script src="js/components/dynamic-upgrade.js"></script>
<script src="js/components/upgrade-container.js"></script>
<script src="js/components/upgrade-tree.js"></script>
<script src="js/components/upgrade-tree-arrow.js"></script>
<script src="js/components/challenge.js"></script>
<script src="js/components/challenge-table.js"></script>
<script src="js/components/current-challenge.js"></script>
<script src="js/components/layer-statistics.js"></script>
<script src="js/components/prestige-layer.js"></script>
<script src="js/components/layer-navigation.js"></script>
<script src="js/components/settings-menu.js"></script>
<script src="js/components/volatility-tab.js"></script>
<script src="js/components/aleph-layer.js"></script>
<script src="js/components/automator.js"></script>
<script src="js/components/automator-tab.js"></script>
<script src="js/components/restack-tab.js"></script>
<script src="js/components/meta-layer.js"></script>
<script src="js/components/news-ticker.js"></script>
<script src="js/components/achievement.js"></script>
<script src="js/components/achievement-tab.js"></script>
<script src="js/components/notification.js"></script>
<script src="js/components/notification-container.js"></script>
<script src="js/components/guide-item.js"></script>
<script src="js/components/guide-tab.js"></script>

<script src="js/data/functions.js"></script>
<script src="js/data/computed.js"></script>
<script src="js/data/game.js"></script>

<script src="js/main.js"></script>

</body></html>

**Comment by Jacorb91** [comment]
<html lang="en"><head>
    <meta charset="UTF-8">
    <title>ඞ-lλγers: layer 1</title>
    <link rel="icon" href="favicon.png">

    <meta name="keywords" content="sussy">
    <meta name="description" content="when the layers are demo 😳😳😳">
    <meta name="image" content="https://jwklong.github.io/sussy-layers/cover.png">
    <meta name="theme-color" content="#F5431F">

    <meta name="og:title" content="sussy layers">
    <meta name="og:type" content="website">
    <meta name="og:description" content="when the layers are demo 😳😳😳">
    <meta name="og:image" content="https://jwklong.github.io/sussy-layers/cover.png">

    <link rel="stylesheet" href="css/main.css">
    <link id="theme" rel="stylesheet" href="css/themes/sussy.css">
</head>
<body>
<div id="loading" style="opacity: 0;">
    <h2><span class="omega">ඞ</span> - l λ γ e r s</h2>
    <p>Loading Savegame...</p>
</div>
<div id="app"><header><nav><div><h2><span class="omega">ඞ</span> - l λ γ e r s</h2> <button><img alt="L" src="images/layers.png" class="inline"> Layers</button> <button><img alt="A" src="images/star.svg" class="inline"> Achievements</button> <button><img alt="G" src="images/information-circle.svg" class="inline"> Guide</button> <button><img alt="C" src="favicon.png" class="inline"> Changelog</button> <button><img alt="S" src="images/settings.svg" class="inline"> Settings</button></div> <div><button disabled="disabled"><span>Reach <span class="resource-name" style="color: rgb(255, 140, 0); text-shadow: currentcolor 0px 0px 0.025em, currentcolor 0px 0px 0.0125em, currentcolor 0px 0px 0em;">☛</span></span></button> <button disabled="disabled"><span>Reach <span class="resource-name" style="color: rgb(230, 255, 0); text-shadow: currentcolor 0px 0px 0.05em, currentcolor 0px 0px 0.025em, currentcolor 0px 0px 0em;">🔫</span></span></button> <button disabled="disabled"><span>Reach <span class="resource-name" style="color: rgb(89, 255, 0); text-shadow: currentcolor 0px 0px 0.075em, currentcolor 0px 0px 0.0375em, currentcolor 0px 0px 0em;">🗡</span></span></button> <button disabled="disabled"><span>Reach <span class="resource-name" style="color: rgb(242, 0, 255); text-shadow: currentcolor 0px 0px 0.225em, currentcolor 0px 0px 0.1125em, currentcolor 0px 0px 0em;"><span>ඞ</span><sup>ඞ</sup></span></span></button>  </div></nav> <div class="news-ticker"><span style="animation-duration: 10.5s;">monke</span></div> <div class="layer-navigation"><button title="<○" class="" style="font-size: 1.25rem;"><span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span> <span class="ordinal" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">#1</span></button></div></header> <main><div class="notification-container"></div> <div><div class="prestige-layer"><p class="resource-display">You have <span>0</span> <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></p>  <div class="resource-button"><button>+1 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></div>  <div class="tabs"><button>Generators</button> <button>Upgrades</button>    <button>Statistics</button> <button>Max All (M)</button></div> <div><table class="generator-table"><thead><th>Generator</th> <th>Amount</th> <th>Buy</th> <th>Buy 10</th></thead> <tr><td>Generator <span style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">0</span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">10 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td> <td><button disabled="disabled">100 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td></tr><tr><td>Generator <span style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">1</span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">100 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td> <td><button disabled="disabled">1,000 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td></tr><tr><td>Generator <span style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">2</span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">1,000 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td> <td><button disabled="disabled">10,000 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td></tr><tr><td>Generator <span style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">3</span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">10,000 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td> <td><button disabled="disabled">100,000 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td></tr><tr><td>Generator <span style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">4</span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">1.00e6 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td> <td><button disabled="disabled">1.00e7 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td></tr><tr><td>Generator <span style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">5</span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">1.00e8 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td> <td><button disabled="disabled">1.00e9 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td></tr><tr><td>Generator <span style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">6</span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">1.00e10 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td> <td><button disabled="disabled">1.00e11 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td></tr><tr><td>Generator <span style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">7</span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">1.00e13 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td> <td><button disabled="disabled">1.00e14 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td></tr><tr><td>Generator <span style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">8</span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">1.00e16 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td> <td><button disabled="disabled">1.00e17 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td></tr><tr><td>Generator <span style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">9</span> <span style="font-size: 70%;">x 1</span></td> <td>0 (0)</td> <td><button disabled="disabled">1.00e19 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td> <td><button disabled="disabled">1.00e20 <span class="resource-name" style="color: rgb(255, 0, 0); text-shadow: currentcolor 0px 0px 0em, currentcolor 0px 0px 0em, currentcolor 0px 0px 0em;">○</span></button></td></tr></table></div>     </div></div>          </main> <footer><div><a href="https://veprogames.github.io/omega-layers" target="_blank">Original Game</a> <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" target="_blank">Website</a></div></footer></div>

<script src="https://polyfill.io/v3/polyfill.min.js?features=Array.prototype.flat"></script>

<script src="js/break_eternity.min.js"></script>
<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/@antimatter-dimensions/notations@1.6.0"></script>


<script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>



<script>
    const INFINITY = new Decimal("1.8e308");
    const INFINITY2 = new Decimal("1.8e30800");
    const INFINITY3 = new Decimal("1.8e308000000");
    var LETTERS = ["○","☛","🔫","🗡","ඞ"];
    var ORDERS = "</-=+x>";
    var GIANTS = ["<span class='flipped-v'>ඞ</span>", "α","β"]
</script>

<script src="js/vocabulary.js"></script>
<script src="js/classes/random.js"></script>
<script src="js/classes/utils.js"></script>
<script src="js/classes/featureunlockmanager.js"></script>

<script src="js/classes/generator.js"></script>
<script src="js/classes/powergenerator.js"></script>
<script src="js/classes/upgrade.js"></script>
<script src="js/classes/challenge.js"></script>
<script src="js/classes/prestigelayer.js"></script>
<script src="js/classes/alephlayer.js"></script>
<script src="js/mods/sabotagelayer.js"></script>
<script src="js/classes/achievement.js"></script>
<script src="js/classes/automator.js"></script>
<script src="js/mods/hacker2.js"></script>
<script src="js/classes/restacklayer.js"></script>
<script src="js/classes/metalayer.js"></script>
<script src="js/classes/notification.js"></script>

<script src="js/components/layer-colored-text.js"></script>
<script src="js/components/layer-locked-button.js"></script>
<script src="js/components/generator.js"></script>
<script src="js/components/generator-table.js"></script>
<script src="js/components/resource-name.js"></script>
<script src="js/components/resource-display.js"></script>
<script src="js/components/upgrade.js"></script>
<script src="js/components/resource-upgrade.js"></script>
<script src="js/components/aleph-upgrade.js"></script>
<script src="js/mods/sabotage-upgrade.js"></script>
<script src="js/components/restack-upgrade.js"></script>
<script src="js/components/dynamic-upgrade.js"></script>
<script src="js/components/upgrade-container.js"></script>
<script src="js/components/upgrade-tree.js"></script>
<script src="js/components/upgrade-tree-arrow.js"></script>
<script src="js/components/challenge.js"></script>
<script src="js/components/challenge-table.js"></script>
<script src="js/components/current-challenge.js"></script>
<script src="js/components/layer-statistics.js"></script>
<script src="js/components/prestige-layer.js"></script>
<script src="js/components/layer-navigation.js"></script>
<script src="js/components/settings-menu.js"></script>
<script src="js/components/volatility-tab.js"></script>
<script src="js/components/aleph-layer.js"></script>
<script src="js/mods/sabotage-layer.js"></script>
<script src="js/components/automator.js"></script>
<script src="js/mods/hacker1.js"></script>
<script src="js/components/automator-tab.js"></script>
<script src="js/mods/hackers-tab.js"></script>
<script src="js/components/restack-tab.js"></script>
<script src="js/components/meta-layer.js"></script>
<script src="js/components/news-ticker.js"></script>
<script src="js/components/achievement.js"></script>
<script src="js/components/achievement-tab.js"></script>
<script src="js/components/notification.js"></script>
<script src="js/components/notification-container.js"></script>
<script src="js/components/guide-item.js"></script>
<script src="js/components/guide-tab.js"></script>
<script src="js/mods/changelog-tab.js"></script>

<script src="js/data/functions.js"></script>
<script src="js/data/computed.js"></script>
<script src="js/data/game.js"></script>

<script src="js/main.js"></script>


</body></html>
